test/e2e-go/e2e_go_olm_test.go

```diff
@@ -48,20 +51,25 @@ var _ = Describe("Integrating Go Projects with OLM", func() {
 			Expect(err).NotTo(HaveOccurred())
 
 			By("building the operator bundle image")
-			// Use the existing image tag but with a "-bundle" suffix.
-			imageSplit := strings.SplitN(tc.ImageName, ":", 2)
-			bundleImage := imageSplit[0] + "-bundle"
-			if len(imageSplit) == 2 {
-				bundleImage += ":" + imageSplit[1]
-			}
-			err = tc.Make("bundle-build", "BUNDLE_IMG="+bundleImage)
+			err = tc.Make("bundle-build", "BUNDLE_IMG="+runBundleImg)
 			Expect(err).NotTo(HaveOccurred())
 
-			if isRunningOnKind() {
-				By("loading the bundle image into Kind cluster")
-				err = tc.LoadImageToKindClusterWithName(bundleImage)
-				Expect(err).NotTo(HaveOccurred())
-			}
+			// bundle image should be present in the remote repository in run bundle
+			// implementation.
+			By("push the image to a remote repository")
+			err = tc.Make("docker-push", "IMG="+runBundleImg)
```

Comment:
I assume you're reverting this change @varshaprasad96?

Reply:
@estroz no, afaik the image has to be pushed otherwise `run bundle` can't seem to pull the code.